### ISSUE TITLE
Logo must have an accessible name 

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#13]
# Logo must have an accessible name [ ❗️ Must ]

## Defect level
Feedback will follow a must, should, consider feedback framework:

- ❗️ **Must for if the feedback must be applied.** Failure to follow this feedback may either cause harm to users, block a disabled person from completing an important task, or result in legal risk (fails to meet WCAG 2.1 AA compliance).
- ⚠️ **Should if the feedback is best practice.** Following this feedback will significantly improve the usability for disabled people or reduce the complexity of accessibility barriers.
- ✔️ **Consider for suggestions or enhancements.** Think of these suggestions as reducing the hundreds of papercuts and microaggressive barriers disabled people may face when navigating the web. Going above and beyond here will make us a leader in accessibility. 

## Description

The DCDW logo is missing alt text (which is 100% understandable as it's probably a placeholder too). Hopefully this ticket can act as a reminder to add "DCDW Home page" to it in the future!

<img width="729" alt="Screen Shot 2022-05-01 at 2 05 23 PM" src="https://user-images.githubusercontent.com/14154792/166158690-ae5280b7-6126-4959-9ea5-c09cff69dda0.png">

## Recommendation

Within the `a` element add:
`aria-label="DC Design Week Homepage"`

<img width="659" alt="Screen Shot 2022-05-01 at 2 11 56 PM" src="https://user-images.githubusercontent.com/14154792/166158923-ca9e9afe-30ec-4f76-9e7b-5121ccf98e90.png">
